src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
+      class="Big--Searchbar"
```

Comment:
Pay attention to the class names. Fix it all over the page 

<img width="1021" alt="Screenshot 2022-11-25 at 11 35 24" src="https://user-images.githubusercontent.com/104769168/203948143-accc1fd1-57d6-4d25-981a-97d8a182f40e.png">